### DISCUSSION TITLE
Creality CR200B with similar DGUS touchscreen

### DISCUSSION THREAD

**Discussion by hieppo** [Desuuuu/klipper#94] [Q&A]
I was able to flash klipper onto this printer using the https://github.com/Klipper3d/klipper repo using the config file from https://github.com/ChewyJetpack/CR-200B-Klipper-Config. Moonraker communicates correctly and works with Fluidd and Octoprint. However, this version currently does not have DGUS support. The Creality board is v4.2.5 and the DGUS touchscreen is data 2021-04-10
I tried your klipper fork.with this printer and nothing seems to work. The screen just stay on the boot logo. When tyring to connect from Fluidd or Octoprint, error message return and not able to connect.
Are these build suppose to be relatively compatble with all the STM controller?

**Comment by Desuuuu** [comment]
STM32 is supported, however your particular version of touchscreen might not be.

**Reply by hieppo** [reply]
Unfortunately, when I tried to build and flash the firmware using the STM32 with the 28kb boot loader, Klipper did not respond for connection. Could this be due to synchronization of the main branch?

**Reply by Desuuuu** [reply]
I doubt it. You can try the latest version on the `dgus-reloaded` branch if you'd like.

If it doesn't work, I suggest trying the new version that's on the `dgus-display` branch. It's a complete rework of the display integration. You can read about the details [here](https://github.com/Desuuuu/klipper/discussions/74).

I don't plan on keeping support for the `dgus-reloaded` branch for much longer.
